pkg/proto/nbcontract/v1/config.proto

```diff
@@ -61,11 +65,11 @@ message Configuration {
   bool needs_containerd = 39; //will be removed later
   bool is_krustlet = 40; //can be removed, this only depends on workload runtime, which is a better contract input
   bool needs_docker_login = 41; //useless, will remove
-  FeatureState ipv6_dual_stack_enabled = 42; //seems very specific to RP, discuss with Alex wrt Karpenter
-  string outbound_command = 43; //this is const depending on cloud, can be removed later with smart refactoring
-  bool enable_unattended_upgrade = 44; //seems the current API is messed up with naming, fix soon
-  bool ensure_no_dupe_promiscuous_bridge = 45; //this is little weird, check cse_cmd.sh for actual condition
-  CustomSearchDomain custom_search_domain = 46;
+  optional FeatureState ipv6_dual_stack_enabled = 42; //seems very specific to RP, discuss with Alex wrt Karpenter
+  optional string outbound_command = 43; //this is const depending on cloud, can be removed later with smart refactoring
```

Comment:
Consider grouping cloud-specific config

Reply:
Will keep this in mind while working on the integration in the coming PRs. I am also taking this chance to understand further what groups should be created and categorized.